internal/server/v1/firehose/actions.go

```diff
@@ -64,7 +64,10 @@ func (api *firehoseAPI) handleScale(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *firehoseAPI) handleStart(w http.ResponseWriter, r *http.Request) {
-	var reqBody struct{}
+	var reqBody struct {
+		StopTime *time.Time `json:"stop_time"`
```

Comment:
Is it `stop_time` or `stoptime`?

Reply:
stop_time, same as https://github.com/goto/dex/blob/96920cf893abab96714bd0d9e58288690e927d28/generated/models/firehose_config.go#L44

Reply:
Also I can't see any logic where we advance the stop time by 24 hours, are we expecting to get as-is only?

Reply:
Keeping the stoptime as 24 hours is our specific need. So, I was in favour of this time being set by the console while making request to dex. Dex API shall allow any time sent to it as payload.
WDYT? @mabdh @sudheerpal @AkarshSatija

Reply:
make sense

Reply:
@ishanarya0  putting it here, means that anyone hitting dex can alter the stop time. Is this the expected behaviour @sudheerpal ?

If the logic to set 24 hours is set on Dex, then we only need to make sure dex sends `stop_time` to entropy without needing to introduce the new field on dex start firehose API.

Reply:
Yeah, anyone who would have edit rights on the firehose can do that. Do we want to prevent this for some reason?
Yeah, this is a tradeoff. We can either keep this API generic or make it more convinient for our use case. 

Thoughts @sudheerpal @AkarshSatija ?

Reply:
Logic of 24 hr stop_time is only applicable to log sink type. We can put this logic on dex.